module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupMember.md

```diff
@@ -55,28 +55,32 @@ To list members of a hidden group, the `Member.Read.Hidden` permission is also r
 
 ## Examples
 
-### Example 1: Get a group member by ID
+### Example 1: Retrieve and Select Group Member Properties
 
 ```powershell
-Connect-Entra -Scopes 'GroupMember.Read.All'
```

Comment:
This line shouldn't be removed.

Reply:
Great catch! @alexandair. I will add it again.